src/chat.cpp

```diff
@@ -1294,10 +1294,11 @@ void chat_FindValidReceiver( void )
 //
 // [AK] Allows players (or the server) to send private messages to other players.
 //
-void chat_PrivateMessage( FCommandLine &argv, const ULONG ulReceiver, const bool bServerSelected )
+void chat_PrivateMessage( FCommandLine &argv, const ULONG ulReceiver )
 {
 	ULONG		ulIdx;
 	FString		ChatString;
+	const bool	bServerSelected = ulReceiver;

```

Comment:
Shouldn't this be `const bool bServerSelected = (ulReceiver == MAXPLAYERS);`?

Reply:
Yes, I missed that (really stupid mistake). :(

Reply:
Thanks!